cran-comments.md

```diff
@@ -1,25 +1,19 @@
-# Patch / Resubmission
-This is a patch to add a java version check as required by the CRAN team (mail from Brian Ripley).
-in this resubmission I have
-
-* skipped an example that uses excessive CPU
-
 # Test Environments
 
-* Ubuntu 18.04 on Travis CI
-* Ubuntu 16.04 on Travis CI
-* CRAN winbuilder
-* Ubuntu Linux 16.04 LTS, R-release, GCC on r-hub
+* Ubuntu 18.04 on Github Actions
+* Ubuntu 16.04 on Github Actions
+* Windows 10 (2019) on Github Actions
+* macOS 10.15 on Github Actions
 
 # R CMD check results
 
 There were no ERRORs or WARNINGs
 
 There was 1 NOTE:
 > checking installed package size ... NOTE
-  installed size is 28.8Mb
-  sub-directories of 1Mb or more:
-    java  26.2Mb
+    installed size is 36.3Mb
+    sub-directories of 1Mb or more:
+      java  34.0Mb
 
 Justification: XLConnect uses a java component which we maintain in a separate project, as well as Apache POI 4.1.1 and
```

Comment:
Apache 4.1.2

Should we mention here that dependencies are download on install? Should be clear though since the submitted tarball will be small.